apps/balancer-tools/src/app/apr/(utils)/getFilteredApiUrl.ts

```diff
@@ -67,9 +70,36 @@ export default function getFilteredDateApiUrl(
     .map(([key, value]) => `${key}=${value}`)
     .join("&");
 
-  return `${BASE_URL}/apr/api?startAt=${formatDateToMMDDYYYY(
-    startAt,
-  )}&endAt=${formatDateToMMDDYYYY(endAt)}${
+  return `startAt=${formatDateToMMDDYYYY(startAt)}&endAt=${formatDateToMMDDYYYY(
+    endAt,
+  )}&${params}`;
+}
+
+export function generateApiUrlWithParams(
+  startAt: Date,
+  endAt: Date,
+  searchParams?: SearchParams | null,
+  poolId?: string,
+) {
+  const queryParams = generateQueryParams(startAt, endAt, searchParams);
+  return `${BASE_URL}/apr/api?${queryParams}${
     poolId ? `&poolId=${poolId}` : ""
-  }&${params}`;
+  }`;
+}
+
+export function generateRedirectUrlWithParams(
+  startAt: Date,
+  endAt: Date,
+  searchParams?: SearchParams | null,
+  poolId?: string,
+) {
+  const queryParams = generateQueryParams(startAt, endAt, searchParams);
+  if (poolId) {
+    const network = networkFor(new Pool(poolId).network);
```

Comment:
if this pool doesn't have a gauge it won't have a network here, right?

Reply:
All pools have network, regardless of gauge, We add it on the pre-build.js [link to file](https://github.com/bleu-fi/balancer-tools/blob/main/apps/balancer-tools/src/data/pools-without-gauge.json#L3).